tests/linearGaussian_snpe_test.py

```diff
@@ -524,20 +525,24 @@ def simulator(theta):
     # Test whether SNPE works properly with structured z-scoring.
     net = posterior_nn("maf", z_score_x="structured", hidden_features=20)
 
-    inference = SNPE_C(prior, density_estimator=net, show_progress_bars=False)
+    inference = SNPE_C(prior, density_estimator=net, show_progress_bars=True)
 
     # We need a pretty big dataset to properly model the bimodality.
     theta, x = simulate_for_sbi(
-        simulator, prior, num_simulations, simulation_batch_size=num_simulations
+        simulator,
+        prior,
+        num_simulations,
+        simulation_batch_size=10,  # use small batch size to ensure bimodality
```

Comment:
I don't get the comment. Why do we need a small batch size?

Reply:
the simulator above decides for each batch or theta, whether to sample from mode one or mode two. if we pass all thetas at once (`simulation_batch_size=num_simulations`), we get data only from one mode.

Reply:
I see, but this is then kinda a bug in the simulator (which is not "vectorized"). 

But I guess it's fine; when seeing the simulator code, the comment becomes clear (which was not rendered in review).

Reply:
yes, you are right, the using the simulator with batch size larger one is a bug then. will push a fix.